apps/web/vibes/soul/primitives/blog-post-carousel/index.tsx

```diff
@@ -7,21 +9,23 @@ import {
   CarouselScrollbar,
 } from '@/vibes/soul/primitives/carousel';
 
-interface Props {
-  className?: string;
-  blogPosts: BlogPostCardBlogPost[];
-  scrollbarLabel?: string;
-  previousLabel?: string;
-  nextLabel?: string;
-}
-
 export function BlogPostCarousel({
   className,
   blogPosts,
   scrollbarLabel,
   previousLabel,
   nextLabel,
-}: Props) {
+}: {
+  className?: string;
+  blogPosts: Array<
+    ComponentPropsWithoutRef<typeof BlogPostCard>['blogPost'] & {
+      id: string;
+    }
+  >;
```

Comment:
I don't like this. I feel like we should export the props instead and alias them when imported. Or decouple the props and define them here